### DISCUSSION TITLE
I am getting a 503 server unavailable error while editing configuration

### DISCUSSION THREAD

**Discussion by himanshu046** [airbytehq/airbyte#29209] [Connector Questions]
in background logs 
1) WARN i.t.i.r.GrpcSyncRetryer(retry):56 - Retrying after failure
io.grpc.StatusRuntimeException: DEADLINE_EXCEEDED: deadline exceeded after 9.999962978s. [closed=[], open=[[remote_addr=daspire-temporal/172.30.0.2:7233]]]
	at io.grpc.stub.ClientCalls.toStatusRuntimeException(ClientCalls.java:262) ~[grpc-stub-1.44.1.jar:1.44.1]
	at io.grpc.stub.ClientCalls.getUnchecked(ClientCalls.java:243) ~[grpc-stub-1.44.1.jar:1.44.1]
	
sometime

2) WARN i.t.i.w.WorkflowWorker$TaskHandlerImpl(handle):256 - Workflow task failure during replying to the server. startedEventId=0, WorkflowId=connection_manager_166f6ce7-ded3-46d5-a5b6-2fc56abd7f3f, RunId=cb4e0cab-a935-48a8-bac6-ea7e7c492bbd. If seen continuously the workflow might be stuck.
io.grpc.StatusRuntimeException: NOT_FOUND: query task not found, or already expired

**Comment by marcosmarxm** [comment]
Both are warning. What version of Airbyte are you running? Did you upgrade from a previous version or a fresh start?

[a comment by an automated account is not shown]
